source/administration/changelog.md

```diff
@@ -4,6 +4,8 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+**Note:** **Upgrade to server version v5.19 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.9 has ended and upgrading to server ESR v5.19 or later is required for improved security, performance, mobile app compatibility, and user experience. See [this blog post](https://mattermost.com/blog/support-for-esr-5-9-has-ended/) for more details on why you should upgrade and how to upgrade in quick, simple steps.
```

Comment:
Makes sense to have this here, but people who don't upgrade don't look a the changelog so it may not have much impact.

Reply:
Is it in the upgrade notes as well?

Reply:
Added to upgrade notes.